Library/Homebrew/cask/lib/hbc/cli/abstract_command.rb

```diff
@@ -49,24 +49,18 @@ def casks(alternative: -> { [] })
         casks = args.empty? ? alternative.call : args
         @casks = casks.map { |cask| CaskLoader.load(cask) }
       rescue CaskUnavailableError => e
-        reason = [e.reason, suggestion_message(e.token)].join(" ")
+        reason = [e.reason, *suggestion_message(e.token)].join(" ")
         raise e.class.new(e.token, reason)
       end
 
       def suggestion_message(cask_token)
-        exact_match, partial_matches = Search.search(cask_token)
+        matches, = Search.search(cask_token)
 
-        if exact_match.nil? && partial_matches.count == 1
-          exact_match = partial_matches.first
-        end
-
-        if exact_match
-          "Did you mean “#{exact_match}”?"
-        elsif !partial_matches.empty?
+        if matches.one?
+          "Did you mean “#{matches.first}”?"
+        elsif matches.any?
```

Comment:
Is `.any?` and `! .empty?` identical performance wise? I was under the impression the latter was more performant (although that could be a Railism, creeping in)

Reply:
[Seems like it](https://mensfeld.pl/2015/06/empty-vs-blank-vs-any-why-you-should-not-use-any-to-test-if-theres-anything-in-the-array/), I didn't know about that.